src/models/dataset-config.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ * Configuration for a selectable dataset
+ */
+export interface DatasetConfig {
```

Comment:
Probably not something we need to worry about right now, but at some point I think we'll be dealing with other datasets (pin dataset, datasets for the graphs), so we might want to use a more specific term for these neo datasets.

Reply:
Yeah good point. I'll rename if I'm working in that code again or feel free if you get there before me.